test/integration/advertise_change_test.lua

```diff
@@ -154,3 +154,42 @@ function g.test_2pc()
         }},
     })
 end
+
+function g.test_failover()
+    -- Test for https://github.com/tarantool/cartridge/issues/1029
+
+    local ok, err = g.A1.net_box:call(
+        'package.loaded.cartridge.failover_set_params',
+        {{mode = 'eventual'}}
+    )
+    t.assert_equals({ok, err}, {true, nil})
+
+    local ok, err = g.B1.net_box:call(
```

Comment:
One of the steps to reproduce the two leaders problem is to restart a leader with a different `advertise_uri` after eventual failover is enabled. Why is it omitted here?

Reply:
It's already running a different `advertise_uri`. See `g.before_all`